core/templates/dev/head/forms/formBuilder.js

```diff
@@ -637,10 +637,10 @@ oppia.factory('rteHelperService', [
 oppia.config(['$provide', function($provide) {
   $provide.decorator('taOptions', [
       '$delegate', '$modal', '$timeout', 'focusService', 'taRegisterTool',
-      'rteHelperService',
+      'rteHelperService', 'explorationContextService',
```

Comment:
OK. Just to confirm, you referring to the `{{ include_js_file }}` stuff in the html files right?